python/cudf/cudf/core/column/numerical.py

```diff
@@ -125,6 +125,32 @@ def __contains__(self, item: ScalarLike) -> bool:
         # TODO: Use `scalar`-based `contains` wrapper
         return self.contains(as_column([search_item], dtype=self.dtype)).any()
 
+    @property
+    def values(self) -> cp.ndarray:
+        """
+        Return a CuPy representation of the NumericalColumn.
+        """
+        if is_pandas_nullable_extension_dtype(self.dtype):
+            dtype = getattr(self.dtype, "numpy_dtype", self.dtype)
+        else:
+            dtype = self.dtype
```

Comment:
Minor suggestion, perhaps a bit simpler.
```suggestion
        dtype = self.dtype
        if is_pandas_nullable_extension_dtype(dtype):
            dtype = getattr(dtype, "numpy_dtype", dtype)
```